examples/customsortfiltermodel/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	qt "github.com/mappu/miqt/qt6"
+	"os"
+)
+
+func GetTreeView(parent *qt.QWidget) *qt.QTreeView {
+	treeView := qt.NewQTreeView(parent)
+	treeView.SetAlternatingRowColors(true)
+	treeView.SetSortingEnabled(true)
+	treeView.SetColumnWidth(0, 300)
+	treeView.SetContentsMargins(0, 0, 0, 0)
+	treeView.Header().SetDefaultSectionSize(200)
+	treeView.Header().SetStretchLastSection(true)
+	treeView.SetSelectionMode(qt.QAbstractItemView__ExtendedSelection)
+
+	return treeView
+}
+
+func PopulateData(rootItem *TreeItem) {
+	cellData := make([]*TreeCell, 0)
+	cellData = append(cellData, NewTreeCell(qt.DisplayRole, *qt.NewQVariant11("Big File")))
+	cellData = append(cellData, NewTreeCell(qt.DisplayRole, *qt.NewQVariant11("10.7 MiB")))
+	item := NewTreeItem(cellData, rootItem, false)
+	err := item.SetData(1, qt.NewQVariant11("11225400"), qt.UserRole)
```

Comment:
The sort size is stored in a QVariant as a string. Then later, we parse it. It might be simpler to use the QVariant constructor that stores an int directly,

Reply:
Good call